HotSpot/View Controllers/BookingViewController.m

```diff
@@ -34,12 +34,31 @@ @implementation BookingViewController
     BOOL pickingStartTime;
     NSIndexPath *startIndexPath;
     NSDateFormatter *formatter;
+    UIColor *themeRedColor;
+    BOOL viewDidLayoutSubviewsForTheFirstTime;
 }
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    viewDidLayoutSubviewsForTheFirstTime = YES;
+    
+    themeRedColor = [UIColor colorWithDisplayP3Red:0.89406615499999997
```

Comment:
if this exact color is reused a lot, it should probably live in a utilities file in a helper C function. Ask Aaron about the C functions if you're not sure!